corehq/apps/users/bulkupload.py

```diff
@@ -487,12 +487,14 @@ def is_password(password):
                     if language:
                         user.language = language
                     if email:
-                        email_validator = EmailValidator()
                         try:
-                            email_validator(email)
-                            user.email = email
+                            validate_email(email)
                         except ValidationError:
                             raise UserUploadError(_("User has an invalid email address"))
+
+                        if not email.islower():
+                            raise UserUploadError(_("Email address must be lower case: {}".format(email)))
```

Comment:
i originally looked into doing that and i think in 99.9% of the time that's fine. there were some people noting that in rare situations caps could make a difference. i think looking at this again, it's probably not worth having people go through the hassle for that 0.001% change